source/particle/interpolator/bilinear_least_squares.cc

```diff
@@ -51,6 +54,18 @@ namespace aspect
         AssertThrow(selected_properties.n_selected_components(n_particle_properties) == 1,
                     ExcNotImplemented("Interpolation of multiple components is not supported."));
 
+        if (use_global_valued_limter)
+          {
+            AssertThrow(global_minimum_particle_properties.size() == n_particle_properties,
+                        ExcMessage("Make sure that the size of 'Global minimum particle property' is equivalent to the number "
+                                   "of particle properties."));
+
+            AssertThrow(global_maximum_particle_properties.size() == n_particle_properties,
+                        ExcMessage("Make sure that the size of 'Global maximum particle property' is equivalent to the number "
+                                   "of particle properties."));
+
+          }
```

Comment:
Please move this whole block into the `parse_parameters` function. You will likely need to get the number of particle properties in a different way (they will not be initialized at that point), but something like the following should work:
```
const Postprocess::Particles<dim> *particle_postprocessor = this->template find_postprocessor<Postprocess::Particles<dim> >();

const unsigned int n_particle_properties = particle_postprocessor->get_particle_world()->get_property_manager->get_data_info()->n_components();
```

This will be both faster (this function is called a lot), and safer (usually we check input parameters right after reading them to not keep wrong values around).